model/gen-ai/spans.yaml

```diff
@@ -16,6 +16,12 @@ groups:
         requirement_level: required
       - ref: gen_ai.request.max_tokens
         requirement_level: recommended
+      - ref: gen_ai.request.min_response_candidates
```

Comment:
please generate MD files - https://github.com/open-telemetry/semantic-conventions/blob/main/CONTRIBUTING.md#2-update-the-markdown-files

Reply:
Thanks. I'm currently blocked on tooling:

https://github.com/open-telemetry/semantic-conventions/pull/1926

If you're able to help me get that in so that I can run the `make` workflow, I would appreciate that.

Reply:
Thank for helping get the tooling dependency through. Updated the MD files using that process.